ansible/playbooks/tools/upgrade-reboot.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+# upgrade - upgrade packages which are safe to be updated.
+# This means no packages will be removed.
+# The machine is rebooted if necessary. Skip the reboot
+# tag to avoid that.
+# Copyright (C) 2020 Ellen Papsch <ellenpapsch@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Please move the SPDX identifier and Copyright string to the top of the playbook, under the `---` line.